opentech/apply/activity/messaging.py

```diff
@@ -108,13 +108,13 @@ class ActivityAdapter(AdapterBase):
     messages = {
         MESSAGES.TRANSITION: 'Progressed from {old_phase.display_name} to {submission.phase}',
         MESSAGES.NEW_SUBMISSION: 'Submitted {submission.title} for {submission.page.title}',
-        MESSAGES.EDIT: '{user} edited',
+        MESSAGES.EDIT: 'Edited',
         MESSAGES.UPDATE_LEAD: 'Lead changed from {old.lead} to {submission.lead}',
         MESSAGES.DETERMINATION_OUTCOME: 'Sent a determination. Outcome: {submission.determination.clean_outcome}',
```

Comment:
This crossed my mind a few times: it would be nice if the log items for reviews/determinations would link to the actual review/determination.. Nice to have, not a must.

Reply:
Yeah, that's on my "nice to have" list